Formula/q/qshell.rb

```diff
@@ -0,0 +1,27 @@
+class Qshell < Formula
+  desc "Shell Tools for Qiniu Cloud"
+  homepage "https://github.com/qiniu/qshell"
+  url "https://github.com/qiniu/qshell/archive/refs/tags/v2.14.0.tar.gz"
+  sha256 "1bb1eb9c7c71c0af7714fa49b04b5e27c851e49a0f8fb0360254cb3c9f4ee472"
+  license "MIT"
+  head "https://github.com/qiniu/qshell.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -extldflags '-static' -X 'github.com/qiniu/qshell/v2/iqshell/common/version.version=v#{version}'"), "./main"
+
+    generate_completions_from_executable(bin/"qshell", "completion")
+  end
+
+  test do
+    output = shell_output "#{bin}/qshell -v"
+
+    assert_match "qshell version v#{version}", output
```

Comment:
We need a more substantial test. Checking the version is fine, and should remain, but we also need something that verifies the functionality of the app so we can make sure it built properly.